### ISSUE TITLE
Allow ordering by document attributes, not just creation_date in project embeds

### ISSUE THREAD

**Issue by duckduckgrayduck** [MuckRock/documentcloud-frontend#286]
Currently, in project embeds, you can only order by creation_date like so:
https://embed.documentcloud.org/projects/illinois-tifs-2020-210985/?embed=1&order=created_at
you cannot however order by title, despite being able to sort by title in the search view.

**Comment by evilevel** [comment]
Yes, would be great to have the 'sort by title' back! At our website SpecialBranchFiles.uk, built before this option was thrown out, the file overviews still have 'in chronological order' in the heading. And I certainly remember accustoming all titles with dates in reverse to allow the listing turn out this way!
Anyway, we're now building a new and large website, and we do intend to use the embedding function again!
To align the options with the search function's document attributes would be great!
eveline

**Comment by duckduckgrayduck** [comment]
This has come up again on News Nerdery

**Comment by duckduckgrayduck** [comment]
This is captured in #209 already.
